tests/fail/stacked_borrows/fnentry_invalidation.stderr

```diff
@@ -0,0 +1,28 @@
+error: Undefined Behavior: attempting a read access using <TAG> at ALLOC[0x0], but that tag does not exist in the borrow stack for this location
+  --> $DIR/fnentry_invalidation.rs:LL:CC
+   |
+LL |         let _oof = *z;
+   |                    ^^
+   |                    |
+   |                    attempting a read access using <TAG> at ALLOC[0x0], but that tag does not exist in the borrow stack for this location
+   |                    this error occurs as part of an access at ALLOC[0x0..0x4]
+   |
+   = help: this indicates a potential bug in the program: it performed an invalid operation, but the Stacked Borrows rules it violated are still experimental
+   = help: see https://github.com/rust-lang/unsafe-code-guidelines/blob/master/wip/stacked-borrows.md for further information
+help: <TAG> was created by a SharedReadWrite retag at offsets [0x0..0x4]
+  --> $DIR/fnentry_invalidation.rs:LL:CC
+   |
+LL |     let z = &mut x as *mut i32;
+   |             ^^^^^^
+help: <TAG> was later invalidated at offsets [0x0..0x4] by a Unique FnEntry retag
+  --> $DIR/fnentry_invalidation.rs:LL:CC
+   |
+LL |     x.do_bad();
```

Comment:
So the point of the test is that it points at the call site? Please add a comment in the test explaining that.